_protocols/new-dna-extraction.md

```diff
@@ -20,105 +20,113 @@ Needed consumables:
 
     ![](/images/protocols/beer-dna-extraction/buffer_collection.svg){: width="25%"}
 
-8. Mix by pipetting to resolve the pellet (Aspirate and pull out the liquid a couple of time with the pipette. You will see that the pellet will go into solution and disappear.) Afterwards, no solid phase should be visible and the solution should turned into a brownish color.
+8. Mix by pipetting to resolve the pellet (Aspirate and pull out the liquid a couple of time with the pipette. You will see that the pellet will go into solution and disappear.) Afterwards, no solid phase should be visible and the solution should turn into a brownish color.
     
    ![](/images/protocols/beer-dna-extraction/suspend_pellet.svg){: width="85%"}
 
-9. Fill up to 20ml
+9. Fill up to 20ml whith the TrisHCl-buffer 
 10. Centrifuge 4000 rpm 10 min 4°C
 11. Discard supernatant
 12. Resuspend the cells (with ca. 1ml 1M TrisHCl buffer pH 7.4) 
-13. Transfer the solution into a 1.5 ml Eppendorf tube.
+13. Transfer the solution into a 1.5 ml Eppendorf tube (eppi).
    ![](/images/protocols/beer-dna-extraction/transfer_to_eppendorf.svg){: width="20%"}
-
-14. Weigth pellets (fresh empty 1.5ml eppi as tara): 6 eppis, weights between 30mg and 60mg 
-15. Pool pellets from two eppis to achieve ca. 70-90mg pellet per eppi (final # of eppis: 3)
+14. Centrifuge 4000 rpm 10 min 4°C
+15. Discard supernatant
+16. Weigh each of the 6 eppy pellets (use empty 1.5ml eppi as tara): weights of one pellet between 30mg and 60mg 
+17. Resolve the pellets by adding 500 μL and pipet up and down 
+18. Combine the solution of two eppis into one to achieve ca. 70-90mg pellet per eppi (final # of eppis: 3)
+19. Centrifuge 4000 rpm 10 min 4°C
+20. Discard supernatant
 
 
 
 ### STEP 2: Break-down the yeast cell wall – first round
-e now want to get the DNA out the yeast. The DNA is well protected by the membrane of the nucleus and the membrane of the cell. We need to break the membrane of the yeast and then the menbrane of the nucleus.
+Now, we want to get the DNA out the yeast. The DNA is well protected by the cell wall and the membrane of the nucleus. We need to break the membrane of the yeast and then the menbrane of the nucleus.
 
 ![](/images/protocols/beer-dna-extraction/yeast_cell.svg){: width="50%"}
 
 *A yeast cell - Frankie Robertson, CC ASA, [Wikimedia](https://en.wikipedia.org/wiki/File:Yeast_cell_english.svg)*
 
-Here: incubation times can be prolonged for difficult to lyse organisms
-- Suspend cells in an appropriate amount of the Y-PER Reagent. Scale the amount of Y-PER Reagent accordingly, maintaining a ratio of 8μL/1mg pellet. 
-	- We assumed all pellets correspond to 80 mg and added 640µl Y-PER
-- Mix by pipetting up and down until the mixture is homogenous
+- Suspend cells:
+ 1. add an appropriate amount of the Y-PER Reagent. Scale the amount of Y-PER Reagent accordingly, maintaining a ratio of 8μL/1mg pellet. 
```

Comment:
Y-PER is a detergent optimized for yeast cell lysis. (https://www.thermofisher.com/order/catalog/product/78991#/78991)